cmd/loadTests.go

```diff
@@ -1261,11 +1248,11 @@ func (h ItsSuccessHandler) HandleSuccess(ctx *JourneyContext, itsName string, ti
 	handleItsSuccess(ctx, itsName, h.Username, h.Application, timeInSeconds)
 }
 
-type CdqSuccessHandler struct{}
+// type CdqSuccessHandler struct{}
 
-func (h CdqSuccessHandler) HandleSuccess(ctx *JourneyContext, cdqName string, timeInSeconds float64) {
-	handleCdqSuccess(ctx, cdqName, timeInSeconds)
-}
+// func (h CdqSuccessHandler) HandleSuccess(ctx *JourneyContext, cdqName string, timeInSeconds float64) {
```

Comment:
Leftover?

Reply:
`cmd/loadTests.go` will be deleted completely in https://github.com/redhat-appstudio/e2e-tests/pull/1163 anyway, so feel free to ignore I would say.

Reply:
Removed.